src/microbe_stage/AgentProjectile.cs

```diff
@@ -4,14 +4,18 @@
 ///   This is a shot agent projectile, does damage on hitting a cell of different species
 /// </summary>
 [JSONAlwaysDynamicType]
-[SceneLoadedClass("res://src/microbe_stage/AgentProjectile.tscn", UsesEarlyResolve = false)]
```

Comment:
Why did you remove this line? I'm pretty close to certain that this will break loading saves.